friends/README.md

```diff
@@ -2,7 +2,7 @@
 
 ## Friends service - Java implementation
 
-This is an implementation of the friends service, which is part of Lightbend Cloudstate Chat sample.
+This is an implementation of the friends service, which is part of Cloudstate Chat sample.
```

Comment:
Hi @retgits and @michaelpnash,

I feel a little bit confused here. Should I use "Akka Serverless Chat sample" or "Cloudstate Chat sample" here?  In the README https://github.com/cloudstateio/samples-java-chat/blob/master/README.md and every other place, I saw it called `CloudState Chat sample`, so I keep "CloudState Chat sample" to make it consistent. Is it correct?

Reply:
Cloudstate Chat sample 👍